<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenU</class>
 <widget class="QDialog" name="MenU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>806</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget {
    background-color: black; /* Фон всего QTabWidget */
}

QTabWidget::pane {
    border: 1px solid gray; /* Граница вокруг содержимого вкладок */
    background-color: black; /* Фон содержимого вкладок */
}

QTabBar::tab {
    color: white; /* Цвет текста вкладки */
    background-color: black; /* Фон вкладки */
    border: 1px solid white; /* Граница вкладки */
    padding: 10px; /* Отступы внутри вкладки */
    margin: 0px; /* Внешние отступы вкладки */
}

QTabBar::tab:selected {
    background-color: gray; /* Фон выбранной вкладки */
    color: black; /* Цвет текста выбранной вкладки */
}

QTabBar::tab:hover {
    background-color: #333; /* Фон вкладки при наведении */
    color: white; /* Цвет текста при наведении */
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Поиск</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>551</width>
       <height>781</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget {
    background-color: black; /* Фон всего QTabWidget */
}

QTabWidget::pane {
    border: 1px solid gray; /* Граница вокруг содержимого вкладок */
    background-color: black; /* Фон содержимого вкладок */
}

QTabBar::tab {
    color: white; /* Цвет текста вкладки */
    background-color: black; /* Фон вкладки */
    border: 1px solid white; /* Граница вкладки */
    padding: 10px; /* Отступы внутри вкладки */
    margin: 0px; /* Внешние отступы вкладки */
}

QTabBar::tab:selected {
    background-color: gray; /* Фон выбранной вкладки */
    color: black; /* Цвет текста выбранной вкладки */
}

QTabBar::tab:hover {
    background-color: #333; /* Фон вкладки при наведении */
    color: white; /* Цвет текста при наведении */
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Додо Пицца</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>551</width>
         <height>741</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI Semibold</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="title">
        <string>Menu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>541</width>
          <height>701</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Теремок</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>551</width>
         <height>741</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI Semibold</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="title">
        <string>Menu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Coffee Like</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>551</width>
         <height>741</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI Semibold</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="title">
        <string>Menu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Корзина</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>531</width>
       <height>761</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semibold</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
color:rgb(255, 255, 255);
}</string>
     </property>
     <property name="title">
      <string>Корзина</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>531</width>
        <height>721</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Личный Кабинет</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>531</width>
       <height>771</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semibold</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
color:rgb(255, 255, 255);
background-color:rgb(0, 0, 0);
}</string>
     </property>
     <property name="title">
      <string>Profile</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>261</width>
        <height>150</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>28</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 28pt &quot;Segoe UI&quot;;
color:rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>24</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 24pt &quot;Segoe UI&quot;;
color:rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>История заказов</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>531</width>
        <height>521</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">  QTableView {
  color: rgb(255, 255, 255);
    background-color: rgb(0, 0, 0);
    gridline-color: rgb(128, 128, 128);
    alternate-background-color: rgb(30, 30, 30);
}

QTableView::Header {
    color: rgb(255, 255, 255);
    background-color: rgb(0, 0, 0);
}

QTableView::item:selected {
    color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>204</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color:rgb(255, 255, 255);
background-color:rgb(0, 0, 0);
border: 2px solid rgb(124, 124, 124);
solid: rgb(255, 255, 255);
border-radius:15px;
max-height:30px;
max-width:200px;
min-height:30px;
min-width:200px;
}</string>
      </property>
      <property name="text">
       <string>Setting</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
